.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Feature Request
+about: Suggest a new feature or enhancement for qBittorrent.
+title: ''
+labels: 'Feature request'
+assignees: ''
+---
+
+# Feature request
+
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+-->
+
+- [ ] I have read the **feature request guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good feature request with the required information
+- [ ] I have verified that the **feature** I am requesting is **not available** in the **[latest version](https://www.qbittorrent.org/download.php)**
+- [ ] (optional but recommended) I have verified that the **feature** I am requesting is **not available** in the **[latest CI build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci.yaml?query=branch%3Amaster+event%3Apush)**
+- [ ] I have **checked the [frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and **searched** the issue tracker for similar feature requests (including closed ones) **to avoid posting a duplicate**
+- [ ] This is **not a support issue or question** better suited for either the [discussions section](https://github.com/qbittorrent/qBittorrent/discussions), [forum](https://qbforums.shiki.hu/), or [subreddit](https://www.reddit.com/r/qBittorrent/), and the [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain a suitable solution either
+- [ ] I have filled in all the other required information below
```

Comment:
Is this necessary?

Reply:
Indeed, probably not. Addressed by noting in the comments that the user is expected to fill out all `(type here)` fields.